src/app/pages/MostReadPage/index.jsx

```diff
@@ -133,7 +134,7 @@ const MostReadPage = ({ pageData, mostReadEndpointOverride }) => {
         description={`${header} - ${brandName}`}
         openGraphType="website"
       />
-
+      <LinkedData type="WebPage" seoTitle={header} />
```

Comment:
Is this something that was missed previously, that's unrelated to the AMP page?

Reply:
Yeah I missed this in the initial creation of the most read page, I found I needed through the integration tests. I've confirmed that this returns the correct values with @JonBeeb on Slack
![image (16)](https://user-images.githubusercontent.com/30599794/81302948-f936a180-9072-11ea-929d-7c89544a4db5.png)